front/src/NotificationWindow.js

```diff
@@ -0,0 +1,67 @@
+import React from "react";
+import styled from "styled-components";
+import faTimes from "@fortawesome/fontawesome-free-solid/faTimes";
+import FontAwesomeIcon from "@fortawesome/react-fontawesome";
+
+const Window = styled.div`
+  background-color : ${props => props.backgroundColor}
+  border-style: solid;
+  border-size: 4px;
+  width: 90%;
+  position: relative;
+  margin-left:auto;
+  margin-right:auto;
+`;
+
+const NotificationHeader = styled.p`
+  text-align: center;
+  font-size: 5vmin;
+`;
+
+const NotificationText = styled.p`
+  padding-left: 10px;
+  padding-right: 10px;
+  text-align: center;
+  font-size: 8vmin;
+`;
+
+const NotificationCloseButton = styled.button`
+  border: none;
+  background-color: transparent;
+  position: absolute;
+  top: 0;
+  right: 0;
+  width: 10vmin;
+  height: 10vmin;
+  &:focus {
+    outline: 0;
+  }
+  font-size: 2vmin;
+`;
+
+/**
+ * Creates a notification window
+ * Author: S.A. Tanja
+ *
+ * Props:
+ * - backgroundColor: sets the background color of the notification window
+ * - notificationType: sets the title of the notification window
+ * - onClick: on close function should be passed
+ * - children: the message of the notification window
+ */
+const NotificationWindow = props => {
+  return (
+    <Window backgroundColor={props.backgroundColor}>
```

Comment:
How many colors are there - 2?  I think `isError` prop is better. Let the component render the color on its own.

Reply:
Perhaps discuss this with the rest of the group? Maybe we need more than two colors, i.e. green for confirmation and red for errors.

Reply:
Issue with an isError prop is that it is binary: there are only two allowed values. If we want to add a new notification type, you need to add a new prop, creating an one-hot encoding, which isn't very efficient. Issue with the current implementation is that you need to specify the color when you want to render the notification.

Therefore I suggest to add a type parameter, which is a string containing the exception type OR require the user to contain a Notification with a styled component containing color details.

Reply:
do we need more than two notification types?

Reply:
We have 3 types: 'Everything is working fine', 'Warning (battery low)', 'Error (Disconnected from SRRS)'